CMakeLists.txt

```diff
@@ -650,19 +650,11 @@ set(
 )
 mark_as_advanced(HELICS_CMAKECONFIG_INSTALL_DIR)
 
-set(HELICS_INSTALL_TARGETS helics_base helics_base_includes jsoncpp_lib helics-static)
-if(BUILD_SHARED_LIBS)
-    set(HELICS_INSTALL_TARGETS ${HELICS_INSTALL_TARGETS} helics-shared)
-endif()
-if(BUILD_C_SHARED_LIB)
-    set(HELICS_INSTALL_TARGETS ${HELICS_INSTALL_TARGETS} helicsSharedLib helicsCpp98)
-endif()
-if(BUILD_APPS)
-    set(HELICS_INSTALL_TARGETS ${HELICS_INSTALL_TARGETS} helics_apps)
-endif()
-file(GLOB HELICS_CMAKE_MODULES "config/cmake/*.cmake")
-file(COPY ${HELICS_CMAKE_MODULES} DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
-export(TARGETS ${HELICS_INSTALL_TARGETS}
+# Copy required CMake module files to build tree for importing build tree with find_package
+file(COPY config/cmake/addMPI.cmake config/cmake/FindZeroMQ.cmake DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Why are we copying this file?

Reply:
Is this so it is in a separate location for installation?

Reply:
It’s so that they are in the same location relative to HELICSConfig.cmake as they would be for installation.

Reply:
ok